team/claus-bio.md

```diff
@@ -2,6 +2,13 @@
 
 As a coordinator, Claus both enables our team of stewards in delivering codebase stewardship, as well as supports our members with onboarding and policy.
 
-Previously, Claus has managed public sector open source collaborations, consulted governments on procurement reform, and supported the development of policy and innovation agendas for public sector digitalisation at a European level.
+Claus is fascinated by governance in large-scale complex systems, particularly at the intersection between government innovation, infrastructure, international relations and industrial policy.
 
-Fascinated by governance in large-scale complex systems, Claus has an academic background in international relations, political economy, policy, critical theory and urbanism.
+Previously, Claus was community and project manager for the [SCORE community](https://score.community/) - helping public sector organizations work together openly. He has also consulted governments on both opening up their procurement processes for innovation and future proofing of their core infrastructure. While working at the European Commission, Claus contributed to the development of policy and innovation agendas for public sector digitalisation at a European level.
+
+Claus holds an MSc in Urbanisation and Development from the London School of Economics and Political Science and a BSc in Politics and International Relations from the University of Manchester.
+
+Find out more at:
+
+* [LinkedIn](https://www.linkedin.com/in/clausmullie)

```

Comment:
Sadly we cannot link to LinkedIn from our site as they require users to sign up and sign their terms of service before seeing any content, this also means we cannot test this link with Travis and thus shouldn't publish them.

I suggest removing it.